selfdrive/locationd/torqued.py

```diff
@@ -51,7 +51,7 @@ def add_point(self, x, y):
 
 class TorqueEstimator(ParameterEstimator):
   def __init__(self, CP, decimated=False):
-    self.hist_len = int(HISTORY / DT_MDL)
+    self.hist_len = int(HISTORY / DT_MDL / 5) if decimated else int(HISTORY / DT_MDL)
```

Comment:
`DT_MDL` is incorrect here since we're comparing it with a 100Hz service, so it's only 1 second of history not 5